styles/cspace/RootPage.css

```diff
@@ -46,9 +46,12 @@ button {
 }
 
 button:hover {
-  background-color: highlightColor;
+  background-color: buttonBgHoverColor;
+  color: buttonBgColor;
+  outline: 2px solid buttonBgColor;
 }
 
 button:focus {
-  outline: 2px solid highlightColor;
+  outline-offset: 2px;
+  outline: 1px solid buttonBgColor;
```

Comment:
Let's make the `outline-offset` 1px, and the `outline` width 2px. I think that looks more consistent with the rest of the UI.